example/bufferCopy/src/bufferCopy.cpp

```diff
@@ -283,6 +283,10 @@ auto main()
     alpaka::mem::view::copy(devQueue, deviceBuffer1, hostViewPlainPtr, extents);
     alpaka::mem::view::copy(devQueue, deviceBuffer2, hostBuffer, extents);
 
+    // Depending on the accelerator, the allocation function may use memory
+    // alignment.
+    // This makes it possible for the pitch to be greater than the memory
+    // which is required for the data.
```

Comment:
I think it may be formulated more directly, smth like "It allows the pitch to exceed the data size"

Reply:
Maybe better:
```
Depending on the accelerator, the allocation function may introduce padding between rows/planes of multidimensional memory allocations.
Therefore the pitch (distance between consecutive rows/planes) may be greater than the space required for the data.
```

Reply:
This is a good explanation. I was not sure, if pitch is a common term.